project_task_material_with_sale_timesheet/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# Copyright 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# Copyright 2016-2017 Tecnativa - Vicent Cubells
+# Copyright 2019 Praxya - Juan Carlos Montoya
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Task Material With Sale Timesheet",
+    "summary": "Add compatibility between  project_task_material_stock"
+               "and sale_timesheet module ",
+    "version": "11.0.1.0.0",
+    "category": "Project Management",
+    "website": "https://github.com/OCA/project/",
+    "author": "Tecnativa, Praxya, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "project_task_material_stock",
+        "sale_timesheet",
+    ],
+    "data": [],
```

Comment:
@jcarlosmontoya Not any File then remove it

Reply:
@nikul-serpentcs Thank you, data item?